Переводчик/Переводчик/Program.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+
+
+namespace Переводчик
+{
+    class Program
+    {
+        static void Main()
+        {
+            Console.WriteLine("Введите текст: ");
+            string lineRus = Console.ReadLine();
+
+            string lineCS = alphabetCS(lineRus);
+      
+            Console.WriteLine("Новый текст: "+ lineCS);
+
+            Console.ReadKey();
+        }
+        static string alphabetCS(string str)
+        {
```

Comment:
Ваше решение работает, но много лишнего кода. Вместо того, чтобы дублировать замены для больших и маленьких букв проще перед заменами перевести строку в верхний регистр.
```suggestion
        {
                str = str.ToUpper();
```